services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/IndividualRequestMdbTest.java

```diff
@@ -73,76 +56,80 @@ public void setUp() {
      * @throws JMSException during the test setup
      */
     @Test
-    public void readMessageSuccess() throws JMSException {
+    public void readMessageSuccessTest() throws JMSException {
         individualRequestMdb = new IndividualRequestMdb(mockTargetSubmissionService);
 
         // Create the actual message to send.
         final ISdtMessage sdtMessage = new SdtMessage();
         sdtMessage.setSdtRequestReference("ReadTest1");
 
         // Wrap the message in the ObjectMessage
-        final ObjectMessage objMessage = EasyMock.createMock(ObjectMessage.class);
-        objMessage.setObject((Serializable) sdtMessage);
-        EasyMock.expectLastCall().anyTimes();
-
-        EasyMock.expect(objMessage.getObject()).andReturn((Serializable) sdtMessage);
-        EasyMock.replay(objMessage);
+        final ObjectMessage objMessage = mock(ObjectMessage.class);
+        when(objMessage.getObject()).thenReturn((Serializable) sdtMessage);
 
         mockTargetSubmissionService.processRequestToSubmit(sdtMessage.getSdtRequestReference());
```

Comment:
This line isn't needed when using Mockito - you can just use verify() to verify that the void method was called when checking.

Reply:
Done